rhoso-observability.md

```diff
@@ -0,0 +1,31 @@
r how buffers/cache/shared are represented<br>* underlying metric is in MB, recording rule corrects to bytes</td></tr><tr style="height: 71px"><td dir="ltr" rowspan="3">VM Disk</td><td dir="ltr">Utilization (R/W)</td><td dir="ltr">ceilometer_disk_device_read_bytes<br>ceilometer_disk_device_write_bytes<br>ceilometer_disk_device_read_requests<br>ceilometer_disk_device_write_requests</td><td dir="ltr">bytes<br><br>count</td><td dir="ltr">system.disk.io<br>system.disk.operations<br></td><td dir="ltr">* separate metrics instead of a direction attribute<br>* device attribute is present as a substring of the resource attribute<br></td></tr><tr style="height: 71px"><td dir="ltr">Utilization (Space)</td><td dir="ltr">ceilometer_disk_device_usage<br>ceilometer_disk_device_capacity</td><td dir="ltr">bytes</td><td dir="ltr">system.filesystem.usage<br>system.filesystem.utilization</td><td dir="ltr">* we&#39;ll need to add a calculation for &quot;utilization&quot;<br>* device attribute is present as a substring of the resource attribute<br>* mode, mountpoint, state, and type attributes are all missing</td></tr><tr style="height: 20px"><td dir="ltr">Duration</td><td dir="ltr">ceilometer_disk_device_read_latency<br>ceilometer_disk_device_write_latency</td><td dir="ltr">ns</td><td dir="ltr">system.disk.io_time<br>system.disk.operation_time</td><td dir="ltr">* ceilometer reports this in nanoseconds instead of seconds<br>* separate metrics instead of a direction attribute<br>* device attribute is present as a substring of the resource attribute<br>* unclear if these are io_time (total wall clock time, no parallel accounting) or operation_time (total * operations time where paralel operations are all counted)</td></tr><tr style="height: 69px"><td dir="ltr" rowspan="3">VM Net</td><td dir="ltr">Utilisation</td><td dir="ltr">ceilometer_network_incoming_bytes<br>ceilometer_network_outgoing_bytes<br>ceilometer_network_incoming_packets<br>ceilometer_network_outgoing_packets</td><td dir="ltr">bytes</td><td dir="ltr">system.network.io<br>system.network.packets</td><td dir="ltr">* separate metrics instead of a direction attribute<br>* device attribute is present as a substring of the resource_name attribute<br></td></tr><tr style="height: 69px"><td dir="ltr">Saturation</td><td dir="ltr">ceilometer_network_incoming_packets_drop<br>ceilometer_network_outgoing_packets_drop</td><td dir="ltr">count</td><td dir="ltr">system.network.dropped</td><td dir="ltr">* separate metrics instead of a direction attribute<br>* device attribute is present as a substring of the resource_name attribute</td></tr><tr style="height: 69px"><td dir="ltr">Errors</td><td dir="ltr">ceilometer_network_incoming_packets_error<br>ceilometer_network_outgoing_packets_error</td><td dir="ltr">count</td><td dir="ltr"><br>system.network.errors</td><td dir="ltr">* separate metrics instead of a direction attribute<br>* device attribute is present as a substring of the resource_name attribute</td></tr></tbody></table>
```

Comment:
This is an export/hack-up from my google sheet. Not 100% happy with the result and it will be a PITA to update, but consider it representative for now.